include/zephyr/drivers/spi/spi_nor.h

```diff
@@ -4,8 +4,8 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#ifndef __SPI_NOR_H__
-#define __SPI_NOR_H__
+#ifndef ZEPHYR_INCLUDE_DRIVERS_SPI_NOR_H_
+#define ZEPHYR_INCLUDE_DRIVERS_SPI_NOR_H_
```

Comment:
The header guard macro name should reflect the file path (at least from `DRIVERS_` prefix):
```suggestion
#ifndef ZEPHYR_INCLUDE_DRIVERS_SPI_SPI_NOR_H_
#define ZEPHYR_INCLUDE_DRIVERS_SPI_SPI_NOR_H_